### DISCUSSION TITLE
API to push playing song, time, images and other

### DISCUSSION THREAD

**Discussion by vale-max** [jcorporation/myMPD#1318] [Ideas]
I use the same RPi where I run myMPD as a bluetooth sink using bluealsa and a couple of scripts.
All this is totally independent from myMPD and thus no info is diplayed on the web interface when bluetooth stuff is played.

Would it be possible to add an API set to push a song title, and ideally the playing time too, to be displayed in the web interface when MPD is not playing anything?
I do not need to go as far as being able to control the bluetooth source (play, pause, skip) but having the possibility to display the current song being played would be nice.

**Comment by jcorporation** [comment]
Sorry, but that is no feature that I want to integrate in myMPD.
